### DISCUSSION TITLE
Error while building doc with sphinx-book-theme 1.0

### DISCUSSION THREAD

**Discussion by VForiel** [executablebooks/meta#960] [Q&A] [answered]
Hi!
I just updated sphinx-book-theme to 1.0 but I get an error when I try to build it : 
```
Extension error (pydata_sphinx_theme):
Handler <function update_config at 0x000002AC04977760> for event 'builder-inited' threw an exception (exception: 'icon_links')
```

I tried to search on the changelog what I have to do to adapt my doc to this new version but I didn't see how to solve this problem (and the error is not very explicit so I don't know how to investigate)

**Comment by VForiel** [comment, marked as answer]
Ok, I found out ^^'

The new version require to add `"icon_links": []` (even if we don't want to have any) in the `html_theme_options` dictionary

**Reply by choldgraf** [reply]
Oh that's definitely a bug then...will try to get a patch out!
